backend/src/users/users.controller.ts

```diff
@@ -68,4 +86,50 @@ export class UsersController {
   remove(@Param('id') id: string): Promise<DeleteResult> {
     return this.usersService.remove(+id);
   }
+
+  @Post(':id/avatar')
+  @UseInterceptors(FileInterceptor('file'))
+  @ApiBearerAuth()
+  @ApiResponse({ status: 201, description: 'Success' })
+  @ApiConsumes('multipart/form-data')
+  @ApiBody({
+    description: 'Avatar Photo',
+    type: AvatarUploadDto,
+  })
+  uploadAvatar(
+    @Param('id') id: string,
+    @UploadedFile() file: Express.Multer.File,
+  ) {
+    return this.usersService.updateAvatar(
+      +id,
+      `${this.configService.get<string>('UPLOADS_PATH')}` + file.filename,
```

Comment:
```suggestion
      this.configService.get<string>('UPLOADS_PATH') + file.filename,
```

Reply:
Also, it seems it is possible to have a 500 with a request of this type:
```curl -X 'POST'   'http://localhost:8080/users/1/avatar'   -H 'accept: */*'   -H 'Authorization: Bearer xxxx'   -H 'Content-Type: multipart/form-data'```
Maybe add a check for `file`.